backend/dataall/aws/handlers/glue.py

```diff
@@ -367,19 +365,3 @@ def batch_delete_tables(**data):
                 f'due to: {e}'
             )
             raise e
-
-    @staticmethod
-    @Worker.handler(path='glue.job.runs')
-    def get_job_runs(engine, task: models.Task):
-        with engine.scoped_session() as session:
-            Data_pipeline: models.DataPipeline = session.query(models.DataPipeline).get(
-                task.targetUri
-            )
-            aws = SessionHelper.remote_session(Data_pipeline.AwsAccountId)
-            glue_client = aws.client('glue', region_name=Data_pipeline.region)
-            try:
-                response = glue_client.get_job_runs(JobName=Data_pipeline.name)
-            except ClientError as e:
-                log.warning(f'Could not retrieve pipeline runs , {str(e)}')
-                return []
-            return response['JobRuns']
```

Comment:
Here is a linting problem. Since we have a stage to check linting in the pipeline on AWS, the code won't pass it. I'd recommend to check all of them by `make lint` (there are some issues to datasets, I think they should go away after merging onto a fresh `modularization-main`)